frontend/test/currentUser/currentUser.sagas.spec.ts

```diff
@@ -19,42 +19,44 @@ import * as CurrentUserSaga from '@/v5/store/currentUser/currentUser.sagas';
 import { expectSaga } from 'redux-saga-test-plan';
 import { CurrentUserActions } from '@/v5/store/currentUser/currentUser.redux';
 import { mockServer } from '../../internals/testing/mockServer';
+import {
+	currentUserMockFactory,
+	generateFakeApiKey,
+	generateFakeAvatarFile,
+	generateFakeAvatarUrl,
+	generatePersonlData,
+} from './currentUser.fixtures';
+import { UpdatePersonalData } from '@/v5/store/currentUser/currentUser.types';
+import { spyOnAxiosApiCallWithFile } from '../test.helpers';
+import api from '@/v5/services/api/default';
 
-describe('Current User: sagas', () => {
-
-	const userData = {
-		username: 'Jason',
-		firstName: 'Jason',
-		lastName: 'Voorhees',
-		company: '13DRepo',
-	}
 
-	describe('getProfile', () => {
-		it('should fetch profile data', async () => {
+describe('Current User: sagas', () => {
+	describe('fetchUser', () => {
+		const userData = currentUserMockFactory();
+		it('should fetch user data', async () => {
 			mockServer
-				.get(`/user`)
+				.get('/user')
 				.reply(200, userData);
 
 			await expectSaga(CurrentUserSaga.default)
-				.dispatch(CurrentUserActions.getProfile())
+				.dispatch(CurrentUserActions.fetchUser())
 				.put.like({ // Using like because avatarUrl uses a timestamp
 					action: {
```

Comment:
cant we generate the action using the action creator for fetch usersuccess? If  you need to get rid of something in there you can delete the field.